docs/modules/deployment/pages/backup/debian-ubuntu/backup-debian-ubuntu.adoc

```diff
@@ -0,0 +1,34 @@
+[[backup-debian-ubuntu]]
+.Stop OpenNMS {page-component-title} service
+[source, console]
+----
+sudo systemctl stop opennms
+----
+
+.Create backup folder
+[source, console]
+----
+sudo mkdir /backup
+----
+
+.Backup OpenNMS {page-component-title} binaries, logs, RRD archives, configurations
+[source, console]
+----
+sudo tar --exclude='usr/share/opennms/data/cache' -cvzf /backup/opennms.tar.gz /usr/share/opennms && \
+sudo tar -cvzf /backup/etc-opennms.tar.gz /etc/opennms && \
+sudo tar -cvzf /backup/var-lib-opennms.tar.gz /var/lib/opennms && \
+sudo tar -cvzf /backup/usr-share-java-opennms.tar.gz /usr/share/java/opennms
+----
+
+.Create PostgreSQL database backup
+[source, console]
+----
+sudo runuser -l postgres -c 'pg_dump -W -U opennms -Fc -h localhost -p 5432 -f /backup/opennms_postgres.dmp opennms'
```

Comment:
Recently did a migration to a new server using `pg_dumpall` and it was a lot easier as you don't need to create the user and database when restoring.

Reply:
Will change it accordingly. Thanks!

Reply:
Fixed with https://github.com/OpenNMS/opennms/pull/4484/commits/a19e20a27831c146c9b7355bb8f9c801fc2914e3